spec/tests/angularComponent/ngcOmniboxControllerSpec.js

```diff
@@ -414,6 +416,93 @@ describe('ngcOmnibox.angularComponent.ngcOmniboxController', () => {
     });
   });
 
+  describe('choosing and unchoosing', () => {
+    beforeEach(() => {
+      omniboxController.ngModel = ['one', 'two'];
+    });
+
+    it('should set the ngModel to the choice when multiple is off', () => {
+      omniboxController.choose('three');
+
+      expect(omniboxController.ngModel).toEqual('three');
+    });
+
+    it('should push the choice to an array when multiple is on', () => {
+      omniboxController.multiple = true;
```

Comment:
This really threw me for a loop. So in `beforeEach` you could set `ngModel = ['one', 'two'];` even while `multiple` is off? Then you get to turn `multiple` back on later? This test seems like it has some secret knowledge of idiosyncrasies in the code. Is this really how we want to represent this API for people reading unit tests?

A few solutions come to mind, but I'm not sure if they are solutions because I have forgotten some information about our intent, specifically:

> Can the value of a single selection be an array?

If not: An easy solution (for reader) is to also set `multiple` to true in the `beforeEach()`.
A more comprehensive solution would be to throw an error if ngModel is ever set to an array when multiple is off.

If so: In your "multiple is off" specs, call `.choose(['three', 'four'])` instead of `.choose('three')`.

I'm not sure if any of these things is really that palatable to me, but I do know that swapping between  `multiple` being on and off dynamically when reading these unit tests sequentially is jarring and can cause confusion.